cli/endpoints/batch/deploy-models/heart-classifier-mlflow/deployment-custom/code/batch_driver_parquet.py

```diff
@@ -1,7 +1,11 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license.
+
 import os
 import glob
 import mlflow
 import pandas as pd
+import logging
```

Comment:
system lib imports should come before package imports :D

Reply:
Indeed!